spec/slack-ruby-bot/support/commands_helper_spec.rb

```diff
@@ -81,15 +81,41 @@
     it 'do not show description for command without description' do
       expect(bot_desc).to include("*command_without_description*\n")
     end
+
+    class AnotherBotBase < SlackRubyBot::Bot
```

Comment:
You should be declaring those with `let`, otherwise you're polluting the global namespace. So:

```ruby
let(:another_bot_base) { Class.new(SlackRubyBot::Bot) do ... }
```